basic_datastructure/basic_array/array.PHP

```diff
@@ -0,0 +1,152 @@
+<?php 
+    //------------------------------HOMEWORK 1----------------------------------
+
+    class SimpleArray {
+        /** @var array */
+        public $arr;
+
+        /**
+        * constructor
+        * @param array $arr
+        */
+        public function __construct($arr)
+        {
+            $this->arr = $arr;
+        }
+
+        /**
+        * calculate the sum of elements in array
+        * @return int
+        */
+        public function sum() 
+        {
+            return array_sum($this->arr);
+        }
+
+        /**
+        * calculate the average of elements in array
+        * @return float
+        */
+        public function average() 
+        {
+            return $this->sum() / count($this->arr);
+        }
+
+        /**
+        * calculate the min of elements in array
+        * @return int
+        */
+        public function min() 
+        {
+            $cpyArr = $this->arr;
+            sort($cpyArr);
+            return $cpyArr[0];
+        }
+
+        /**
+        * calculate the max of elements in array
+        * @return int
+        */
+        public function max() 
+        { 
+            $cpyArr = $this->arr;
+            rsort($cpyArr);
+            return $cpyArr[0];
+        }
+
+        /**
+        * print the average, sum, min and max of elements in array
+        * @return void
+        */
+        public function output() 
+        {
+            echo "<br>-sum: " . $this->sum()
+            . "<br>-average: " . $this->average() 
+            . "<br>-min: " . $this->min() 
+            . "<br>-max: " . $this->max() . "<br>";
+        }
+    }
+
+    $arr1 = new SimpleArray(array(5, 12, 17, 9, 3));
+    $arr2 = new SimpleArray(array(13, 4, 8, 14, 1));
+    $arr3 = new SimpleArray(array(9, 5, 3, 7, 21));
+    $arr1->output();
+    $arr2->output();
+    $arr3->output();
+
+    /////////////////////////////////////////////////////////////////////////
+
+    //------------------------------HOMEWORK 2----------------------------------
+
+    $arrA = array(1, 2, 3, 4, 5);
+    $arrB = array(4, 5, 3, 2, 10);
+    $arrC = array_map(function () 

```

Comment:
Try to implement with loop. We don't need to use util method.

Reply:
i used loop for this homework.
Thanks for your comment.